mgit/lib/forkexecutable.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
This file should be called something like `child-process.js` or at least `child.js`. It doesn't `fork` anything, especially not `executable`.